### ISSUE TITLE
BUG: Container element is empty

### ISSUE THREAD

**Issue by KjerstiSteien** [Informasjonsforvaltning/fdk-portal#1843]
### Nåværende oppførsel
Description
Some roles are designed to contain other roles. This element has this type of role, but it doesn't contain any [required-owned elements](https://www.w3.org/TR/wai-aria-1.1/#mustContain).

Learn more
Roles provide information about content structure and how page elements fit together.

Some roles depend on other roles for context. For example, the role listitem can only really be meaningful in the context of a list or group.

Gjelder følgende sider: 
- [x] Etterspørsler: https://my2.siteimprove.com/Inspector/1146729/A11Y/Page?pageId=7301733338&impmd=838C1326AD90AB32DE433803C8F2D3D2#/sia-r68/failed/MzRLZDVqZnc=;NGM5ZjA5ZGE=

- [ ] Bunnen på sider for beskrivelse av informasjonsmodeller, datasett, api-er, begrep, tjenester og hendelser når man ikke er logget inn i datalandsbyen: https://my2.siteimprove.com/Inspector/1146729/A11Y/Page?pageId=6820762212&impmd=838C1326AD90AB32DE433803C8F2D3D2#/sia-r68/failed/YXk1bXVESDM=;Y2UxYmI4Y2E=

- [ ] Mellom lenker under regler på tjenester: https://my2.siteimprove.com/Inspector/1146729/A11Y/Page?pageId=6820762212&impmd=838C1326AD90AB32DE433803C8F2D3D2#/sia-r68/failed/YXk1bXVESDM=;Y2UxYmI4Y2E=

Side i staging som har "Regler": https://staging.fellesdatakatalog.digdir.no/public-services/42e14526-d0b6-3faf-a891-6254b66c0825

### Forventet oppførsel
Forventer at strukturen er i henhold til w3c sine krav. 

### Sjekkliste (Sjekk av etter issue er laget)

- [ ] Lagt til label(s)
- [ ] Lagt i utviklingsbrettet (Project)
